CHANGELOG.unreleased.md

```diff
@@ -17,6 +17,7 @@
 
 - Fix mounting of NFS remote in FreeBSD (PR [#4988](https://github.com/vatesfr/xen-orchestra/issues/4988))
 - [Remotes] Fix "remote is disabled" error on getting the remotes info (commit [eb2f429964d7adc264bf678c37e49a856454388e](https://github.com/vatesfr/xen-orchestra/commit/eb2f429964d7adc264bf678c37e49a856454388e))
+- Remove server's credentials from the error `SESSION_AUTHENTICATION_FAILED` (PR [#4995](https://github.com/vatesfr/xen-orchestra/pull/4995))
```

Comment:
Is it really relevant to add this to the changelog?

If yes, why was it an issue that the credentials where attached to the error?

Reply:
Password attempts can be personal passwords, other servers' password it should not be logged/viewed by other system admins.

Reply:
Your changelog entry does not mention logging.